Northstar.Client/mod/resource/ui/menus/server_browser.menu

```diff
@@ -1428,11 +1428,11 @@ resource/ui/menus/mods_browse.menu
 			navUp BtnFiltersClear
 		}
 
-		BtnServerLatency1
+		BtnServerRegion1
 		{
 			ControlName Label
 			labelText ""
-			classname Serverlatency
+			classname Serverregion
```

Comment:
I would prefer if this was camelCase as opposed to justhavingallthewordslikethiscase

the same goes for all instances of `serverregion`

Reply:
I just changed what already existed by @F1F7Y :>